doc/changelog/10-standard-library/14086-mod-uniform.rst

```diff
@@ -0,0 +1,10 @@
+- **Changed:**
+  set ``n mod 0 = n`` uniformly for ``nat``, ``N``, ``Z``, ``int63``, ``sint63``, ``int31``
+  such that ``m = (m / n) * n + (m mod n)`` holds (also for ``n = 0``)
+
+  .. warning:: code that relies on ``n mod 0 = 0`` will break;
+     for compatibility with both ``n mod 0 = n`` and ``n mod 0 = 0`` you can use
+     ``n mod 0 = ltac:(match eval hnf in (1 mod 0) with |0 => exact 0 |_ => exact n end)``
```

Comment:
BTW, "overusing plain quotes" might be a bad idea. https://github.com/coq/coq/blob/master/doc/sphinx/README.rst#overusing-plain-quotes

Reply:
Yes. Since all of those are just Gallina snippets, `:g:` is appropriate.